### PR TITLE
2024.6 Bitwarden uses camelCase keys instead of PascalCase

### PR DESCRIPTION
**Pull request by OdyX**
This fixes client access to a 1.31 Vaultwarden instance, see:

- https://github.com/dani-garcia/vaultwarden/releases/tag/1.31.0
- "Change API and structs to camelCase by @dani-garcia" in https://github.com/dani-garcia/vaultwarden/pull/4386

### CODE REVIEW

File: src/bitwardentools/client.py [hunk 1]
```diff
@@ -484,7 +484,7 @@ def construct(
             try:
                 object_class_name = jsond["object"]
             except KeyError:
-                object_class_name = jsond["Object"]
+                object_class_name = jsond["object"]
             if object_class_name.lower().startswith("cipher"):
                 try:
                     typ = jsond["Type"]
```
Comment (commonism) [inline src/bitwardentools/client.py:490]: missing - type

Comment (commonism) [reply src/bitwardentools/client.py:490]: below - L.error(f'Unkown cipher {jsond.get("Id", "")}') 
needs to be "id"

**Review by commonism — CHANGES_REQUESTED** [review]

File: src/bitwardentools/client.py (hunk 1 above)
Comment (commonism) [reply src/bitwardentools/client.py:490]: To clarify, missed Type and Id in the related lines below.

**Comment by commonism** [comment]
Required for Vaultwarden 1.31 released 08.07.2024

https://github.com/dani-garcia/vaultwarden/releases/tag/1.31.0
> Change API and structs to camelCase by @dani-garcia in https://github.com/dani-garcia/vaultwarden/pull/4386

File: src/bitwardentools/client.py (hunk 1 above)
Comment (OdyX) [reply src/bitwardentools/client.py]: Right; fixed.

**Comment by commonism** [comment]
@kiorky - this needs your attention, all tooling using bitwarden tools is broken by the API changes and require these updates.

**Comment by commonism** [comment]
Would you accept a patch which makes use of pydantic to process/validate the json?
pydantic is quite a dependency but … besides best in class validation it adds comfort.

e.g. - https://docs.pydantic.dev/latest/concepts/alias/

**Comment by kiorky** [comment]
Hello, i want to thanks you very much as it largely inspired the latest release with my own fix !

**Pull request closed without merging**